src/lib/radial-chart/radial-chart.js

```diff
@@ -245,10 +245,6 @@ class RadialChart extends React.Component {
       return null;
     }
 
-    const {innerWidth, innerHeight} = getInnerDimensions(
```

Comment:
I added that to the example to indicate how it works, but it seems like we could also probably expand the documentation around the usage of margins as well